c7n/resources/rdscluster.py

```diff
@@ -399,6 +401,59 @@ def process(self, clusters):
                 client.exceptions.InvalidDBClusterStateFault)
 
 
+@RDSCluster.action_registry.register('modify-db-cluster')
+class ModifyDbCluster(BaseAction):
+    """Modifies an RDS instance based on specified parameter
+    using ModifyDbInstance.
+
+    'Immediate" determines whether the modification is applied immediately
+    or not. If 'immediate' is not specified, default is false.
+
+    :example:
+
+    .. code-block:: yaml
+
+            policies:
+              - name: disable-db-cluster-deletion-protection
+                resource: rds-cluster
+                filters:
+                  - DeletionProtection: true
+                  - PubliclyAccessible: true
+                actions:
+                  - type: modify-db-cluster
+                    attributes:
+                        CopyTagsToSnapshot: true
+                        DeletionProtection: false
+    """
+
+    schema = type_schema(
+        'modify-db-cluster',
+        immediate={"type": 'boolean'},
+        attributes={'type': 'object'},
+        required=('attributes',))
+
+    permissions = ('rds:ModifyDBCluster',)
+    shape = 'ModifyDBClusterMessage'
+
+    def validate(self):
+        attrs = dict(self.data['attributes'])
+        if 'DBClusterIdentifier' in attrs:
+            raise PolicyValidationError(
+                "Can't include DBClusterIdentifier in modify-db-cluster action")
+        attrs['DBClusterIdentifier'] = 'PolicyValidation'
+        return shape_validate(
+            attrs,
```

Comment:
nit: the parameters to this can all go one a single line.

Reply:
Changed it to a single line